src/commands/deployments/configurePipeline.ts

```diff
@@ -0,0 +1,52 @@
+import * as utils from 'util';
+import * as vscode from 'vscode';
+import { IActionContext } from "vscode-azureextensionui";
+import { SiteTreeItem } from "../../explorer/SiteTreeItem";
+import { WebAppTreeItem } from "../../explorer/WebAppTreeItem";
+import { ext } from "../../extensionVariables";
+import { openUrl } from '../../utils/openUrl';
+
+export async function configurePipeline(context: IActionContext, node: SiteTreeItem): Promise<void> {
+    if (await isAzurePipelinesExtensionInstalled()) {
+        node = await getWebAppNode(context, node);
+        await executeAzurePipelineExtensionCommand(configurePipelineCommand, node);
+    }
+}
+
+async function isAzurePipelinesExtensionInstalled(): Promise<boolean> {
+    const pipelinesExtension = vscode.extensions.getExtension(azurePipelinesExtensionId);
+    if (!pipelinesExtension) {
+        vscode.window.showInformationMessage('Please install/enable `Deploy to Azure` extension to continue.');
```

Comment:
Please use the ext.ui.showWarningMessage like so https://github.com/microsoft/vscode-azureappservice/blob/06bec5e7f2e690b0ec7348e525cf0cc216a34de4/src/commands/deploy/confirmDeploymentPrompt.ts#L29

This will handle if the user cancels the action and you can have an "Install" button that will open the extension in the marketplace rather than forcing the user to change windows.